src/terminal/XTerm.tsx

```diff
@@ -97,7 +97,7 @@ export const XTerm = (props: Props) => {
         });
 
         const disposer = terminal.onData((data) => {
-            props.stream.write(data);
+            props.stream!.write(data);
```

Comment:
Isn't it a bit risky? What if stream is not what we expect?

```suggestion
            props.stream?.write(data);
```

Reply:
I figured it was safe, given the check on line 90 above - this hook only runs if props.stream is set.

Reply:
Ah, yes - didn't see that line